index.html

```diff
@@ -1,140 +1,154 @@
+                    <a href="/"><i class="fas fa-book-reader"></i></a>
+                    <h3>Lecture</h3>
+                    <p>Listen to the speakers from various countries about contributing to open source projects</p>
+                </div>
+                <div class="card-2">
+                    <a href="/"><i class="fas fa-cogs"></i></a>
+                    <h3>Language<br>Exhibition</h3>
+                    <p>Designing same product in different languages. Diversity of knowledge</p>
+                </div>
+                <div class="card-3">
+                    <a href="/"><i class="far fa-comment-dots"></i></a>
+                    <h3>Forum</h3>
+                    <p>Have the chance to share your thoughts and opinions on open source contribution. None coders are
+                        welcome.</p>
+                </div>
+                <div class="card-4">
+                    <a href="/"><i class="fas fa-chalkboard-teacher"></i></a>
+                    <h3>Workshop</h3>
+                    <p>Our workshop will cover a hands-on session on JavaScript and React.</p>
+                </div>
+                <div class="card-5">
+                    <a href="/"><i class="fas fa-handshake"></i></a>
+                    <h3>OS Ignite</h3>
+                    <p>Get opportunity to network with language experts around the world, and an after-event-mentorship
+                    </p>
+                </div>
+                <button class="join-event-btn">Join the OS Global Summit 2021</button>
             </div>
-            <button class="join-event-btn">Join the OS Global Summit 2021</button>
+            <a class="all-program-link" href="/">
+                <p>SEE THE WHOLE PROGRAM</p>
+            </a>
         </div>
-
+    </section>
+    <section class="event-speakes">
         <div class="mobile-third-page">
-            <h2 id="event">Guest Speakers</h2>
-            <div class="image-container">
-                <div class="speakers">
-                    <div class="speaker-image">
-                        <img src="./images/emma-gbenga.png" alt="An Image of the first speaker">
-                    </div>
-                    <div>
-                        <h2>Emma Gbenga</h2>
-                        <h3>Founder Open Source Africa</h3>
-                        <h4>Formal UI designer at Google</h4>
-                    </div>
-                </div>
-                <div class="speakers">
-                    <div class="speaker-image">
-                        <img src="./images/charles-shady.png" alt="An Image of the second speaker">
-                    </div>
-                    <div>
-                        <h2>Charles Shady</h2>
-                        <h3>Full-stack developer</h3>
-                        <h4>Team Lead at Microsoft Inc.</h4>
-                    </div>
-                </div>
+            <h2 id="event">Event Speakers</h2>
+            <div class="image-display-1" id="image-display">
```

Comment:
- [ ] On the mobile view speakers section should look like the design as well

your page
![sp](https://user-images.githubusercontent.com/44745855/142403414-386674d8-81f1-4cd4-a4e3-6b9a13d4f786.png)

original
![p](https://user-images.githubusercontent.com/44745855/142403511-9b6636ef-dae9-4670-bce2-b693e692c786.png)